private/direct.m

```diff
@@ -0,0 +1,750 @@
+function [ret_minval,final_xatmin,history] = direct...
```

Comment:
can we please give this a more informative name?

Reply:
is the algorithm's name ;) was not planned to be integrated with pesto by now, have deleted the file.